region/maridia/outer/Fish Tank.json

```diff
@@ -1990,9 +2006,97 @@
       "link": [7, 6],
       "name": "Base",
       "requires": [
-        "h_canNavigateUnderwater"
+        "Gravity",
+        {"or": [
+          "HiJump",
+          "canSpringBallJumpMidAir",
+          "canDodgeWhileShooting",
+          "canGravityJump",
+          "canCrouchJump",
+          "canDownGrab",
+          {"enemyDamage": {
+            "enemy": "Pink Space Pirate (standing)",
+            "type": "contact",
+            "hits": 1
+          }},
+          {"enemyKill": {
+            "enemies": [["Pink Space Pirate (standing)"]],
+            "explicitWeapons": ["Plasma"]
+          }}
+        ]}
+      ]
+    },
+    {
+      "link": [7, 6],
+      "name": "Suitless Jump Assist",
+      "requires": [
+        "canSuitlessMaridia",
+        {"or": [
+          {"and": [
+            "HiJump",
+            "canSpringBallJumpMidAir"
+          ]},
+          {"and": [
+            "HiJump",
+            "canDodgeWhileShooting"
+          ]},
+          {"and": [
+            "canTrickyJump",
+            "canSpringBallJumpMidAir",
```

Comment:
TrickyJump lets you stand next to the pirate.
- `h_canCrouchJumpDownGrab` can reach the ledge from there.
- Or springball itself can reach the next ledge.